packages/dialtone-vue2/recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue

```diff
@@ -23,9 +23,9 @@
           class="combobox__chip-wrapper"
         >
           <dt-chip
-            v-for="item in selectedItems"
+            v-for="({ item, key }) in selectedItemsWithKeys"
             ref="chips"
-            :key="item"
+            :key="key"
```

Comment:
```
v-for="selectedItem in selectedItems"
:key="getUniqueString(selectedItem)"
```

Im missing sth with this approach?

Reply:
yes 😞 seems to cause some problems when focusing the chips. Unfortunately I couldn't find a good way to fix the main problem and make the tests happy.
With the last solution I tried, everything seems to work fine, but the tests fail (although those cases seem to work fine). I'll have to investigate a bit more to see if the best solution is to just update the tests or provide another solution that makes the tests happy.

Reply:
Ok yes I supposed that test will complain because we will probably need to mock the `getUniqueString` function.
I did a couple of test and seems like this `UNIQUE_ID_COUNTER` of the function is causing troubles. 
If you replace it with something like return '${prefix}${1};'  test passes.

So we just need to add this at the top of the test:

```
import * as utils from '@/common/utils';

vi.spyOn(utils, 'getUniqueString').mockImplementation(() => 'mockedUniqueString');
```

And it should works.

Reference: https://vitest.dev/guide/mocking.html#functions